src/content/docs/data-apis/ingest-apis/event-api/incident-event-rest-api.mdx

```diff
@@ -0,0 +1,312 @@
+---
+title: Incident event REST API
+tags:
+  - Alerts and Applied Intelligence
+  - Applied intelligence
+  - Incident intelligence
+metaDescription: Read about how to report activities from your applied intelligence incident management system to New Relic for advanced correlation.
+---
+
+You can use the [Event API](/docs/data-apis/ingest-apis/introduction-event-api) to report custom applied intelligence incident events to New Relic.
+
+The API is an asynchronous endpoint. This means you can send a large volume of POSTS, reliably, with low-response latency.
+
+## Using the API: an overview [#overview]
+
+Here's the general process for sending custom incident events to New Relic via the incident event API:
+
+1. Generate a [license key](/docs/apis/intro-apis/new-relic-api-keys/#ingest-license-key) for the account you want to report data to.
+2. Before you get started creating custom attributes, review the [Event API's limits and restricted characters](/docs/data-apis/ingest-apis/introduction-event-api/#limits).
+3. Generate the JSON for your incident event that follows our [JSON format guidelines](/docs/data-apis/ingest-apis/introduction-event-api#instrument).
+4. Submit a compressed JSON payload (for example, gzip or deflate) to the HTTPS endpoint using curl in a POST request.
+
+Once you've started sending incident event data to New Relic, we recommend that you set up [NRQL alert conditions](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions) to get notifications about parsing errors.
+
+## Example JSON [#example-json]
+
+Your JSON payload may look something like this example.
+
+```json
+[
+  {
+    "eventType": "NrAiIncidentExternal",
+    "title": "Test",
+    "description": "The latency is above threshold of 500000 MS",
+    "state": "trigger",
+    "source": "luna",
+    "entityName": "testEntity",
+    "entity.guid": "testEntity123",
+    "aggregationTag.serviceId": 5,
+    "aggregationTag.environment": "testing",
+    "aggregationTag.errorId": 10543,
+    "tag.stackTrace": "some stack trace...",
+    "version": 1
+  }
+]
+```
+
+The `eventType` field must use `NrAiIncidentExternal`.
+
+## Send your incident event from the command line [#cli]
+
+Here's an example of how to use curl to send your JSON payload.
+
+```bash
+gzip -c example_incidents.json | curl --data-binary @- -X POST -H "Content-Type: application/json" -H "Api-Key: <var>YOUR_LICENSE_KEY</var>" -H "Content-Encoding: gzip"
+https://staging-insights-collector.newrelic.com/v1/accounts/<var>YOUR_ACCOUNT</var>/events
```

Comment:
happened to notice staging url here

Reply:
That's what they had in the original draft. I'll check on that.

Reply:
Yup, it should be a non-staging URL